docker/README.md

```diff
@@ -51,13 +51,15 @@ Simple stand-alone usage:
 
 To link to another mysql container, for example:
 
-    docker run --rm --name newcentury_mysql -p 3306 \
+    docker run --rm --name mysql \
         -e HUGINN_MYSQL_DATABASE=huginn \
         -e HUGINN_MYSQL_USER=huginn \
         -e HUGINN_MYSQL_PASSWORD=somethingsecret \
         -e HUGINN_MYSQL_ROOT_PASSWORD=somethingevenmoresecret \
```

Comment:
I needed to change those environment variables to the ones you used in the `docker-compose.yml`, am I missing something?